Alicerce/Sources/Logging/Chronicle/Log.swift

```diff
@@ -20,7 +20,7 @@ final class Log {
         return providers.count
     }
     
-    public class func registerProvider(provider: ProviderProtocol) {
+    public class func registerProvider(provider: LogProvider) {
```

Comment:
I also suggested another possibility: `register(_ provider: LogProvider)`

Since we already know the type is `LogProvider`, the parameter name may not be necessary from an API perspective. It does expect the user to use proper variable names, though 😆